hydrolib/core/dflowfm/tim/serializer.py

```diff
@@ -0,0 +1,100 @@
+from pathlib import Path
+from typing import Dict, Iterable, Optional
+
+from hydrolib.core.basemodel import ModelSaveSettings, SerializerConfig
+from hydrolib.core.dflowfm.ini.io_models import Datablock, DatablockRow
+
+
+class TimSerializerConfig(SerializerConfig):
+    max_length_datablock = 10
```

Comment:
Why is this an attribute? The max length should be derived, it should not be set with a constant value I think.

Reply:
Please correct me if I am wrong.
I just think the behaviour will be very weird when some values are longer than 10 chars.